sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -901,6 +901,12 @@ class Analyzer(
       // If the projection list contains Stars, expand it.
       case p: Project if containsStar(p.projectList) =>
         p.copy(projectList = buildExpandedProjectList(p.projectList, p.child))
+      case p @ Project(projectList, _ @ SubqueryAlias(_, view: View))
```

Comment:
Replace `[COL1#10L, COL2#11L]` to `col1#10L,col2#11L`.  thus `DDLPreprocessingUtils.castAndRenameQueryOutput` will take effect and rename column name same to target table  column name.
Then parquet column name same to table column name and we can read it.

![image](https://user-images.githubusercontent.com/5399861/44254005-08891200-a234-11e8-878c-6632a29c8ef1.png)